saleor/graphql/webhook/subscription_types.py

```diff
@@ -846,6 +884,8 @@ class ProductVariantStockUpdated(SubscriptionObjectType, ProductVariantBase):
     )
 
     class Meta:
+        root_type = None
+        enable_dry_run = False
```

Comment:
Why are we changing the meta for `ProductVariantStockUpdated` in PR related to media webhooks?

Reply:
Just for consistency. This are default values, but for visibility, we decided to keep them inside every single subscription type.